cmd/config.go

```diff
@@ -74,34 +151,60 @@ var configRemoveCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		conf := clients.GetConfig(cmd.Context())
 		name, _ := cmd.Flags().GetString("name")
-		cmd.Printf("Removing context: %s\n", name)
+		cmd.Printf("Removing access to project: %s\n", name)
 		for i, c := range conf.Contexts {
 			if c.Name == name {
 				conf.Contexts = append(conf.Contexts[:i], conf.Contexts[i+1:]...)
 			}
 		}
 		viper.Set("contexts", conf.Contexts)
 		if err := viper.WriteConfig(); err != nil {
-			log.Fatalf("Failed to write the config: %s", err)
+			log.Fatalf("Failed to write settings: %s", err)
 		}
-		cmd.Printf("Wrote configuration to file: %s\n", viper.ConfigFileUsed())
+		cmd.Printf("Wrote settings to file: %s\n", viper.ConfigFileUsed())
 	},
 }
 
 var configUseCmd = &cobra.Command{
 	Use:   "use",
-	Short: "Select the default context used",
-	PreRunE: func(cmd *cobra.Command, args []string) error {
-		name, _ := cmd.Flags().GetString("name")
-		return ensureContextExists(cmd.Context(), name)
-	},
+	Short: "Select the default project used",
 	Run: func(cmd *cobra.Command, args []string) {
+		previousContext := viper.Get("current-context")
 		name, _ := cmd.Flags().GetString("name")
+		if name == "" {
+			conf := clients.GetConfig(cmd.Context())
+			cmd.Printf("Known projects:\n")
+			for ixd, c := range conf.Contexts {
+				prefix := fmt.Sprintf("%d:  ", ixd+1)
+				if c.Name == previousContext {
+					prefix = "*   "
+				}
+				if c.Name == c.RemoteName {
+					cmd.Printf("%s%s\n", prefix, c.Name)
+				} else if c.RemoteName == "" {
+					cmd.Printf("%s%s (name unknown)\n", prefix, c.Name)
+				} else {
+					cmd.Printf("%s%s (%s)\n", prefix, c.Name, c.RemoteName)
+				}
+			}
+
+			cmd.Printf("Which project do you want to use [1 .. %d]:\n", len(conf.Contexts))
+			var i int
+			_, err := fmt.Scanf("%d", &i)
+			if err != nil {
+				log.Fatalf("Invalid choice: not a number in range [1 .. %d]", len(conf.Contexts))
+			}
+			if 1 > i || i > len(conf.Contexts) {
+				log.Fatalf("Invalid choice %d: not a number in range [1 .. %d]", i, len(conf.Contexts))
+			}
+			name = conf.Contexts[i-1].Name
+		}
+
 		viper.Set("current-context", name)
```

Comment:
`ensureContextExists(cmd.Context(), name)` should be called to ensure the given name is valid.